src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/EnvelopeCountSummaryRepository.java

```diff
@@ -15,18 +15,26 @@
     @Query(
         nativeQuery = true,
         value = "SELECT\n"
-            + "  container,\n"
-            + "  date,\n"
+            + "  first_events.container AS container,\n"
+            + "  first_events.date AS date,\n"
             + "  count(*) AS received,\n"
-            + "  SUM(CASE WHEN event IN ('DOC_FAILURE', 'FILE_VALIDATION_FAILURE', 'DOC_SIGNATURE_FAILURE') THEN 1 ELSE 0 END) AS rejected\n"
+            + "  SUM(CASE WHEN rejection_events.id IS NOT NULL THEN 1 ELSE 0 END) AS rejected\n"
```

Comment:
`rejection_events.id` could just have number and case/when/then here would become obsolete

Reply:
True, wouldn't be an id, but some count column, always being one. It would work equally well. Would it be clearer? I don't know, because you'd have to check what that column is in the joined select to understand what we're summing. Let's see what others think and I'm ok with either one.